package.json

```diff
@@ -20,7 +20,8 @@
   "bin": "./bin/lint-staged.js",
   "exports": {
     ".": "./lib/index.js",
-    "./package.json": "./package.json"
+    "./package.json": "./package.json",
+    "./bin/lint-staged.js": "./bin/lint-staged.js"
```

Comment:
Would it make sense to export this from just `./bin`?

Reply:
Sounds better!

Reply:
Let's make it so 👍

Reply:
Done.